docs/api/migration-guides/sessions.mdx

```diff
@@ -0,0 +1,94 @@
+---
+title: Execution mode changes
+description: Learn about the changes to execution modes (sessions, batch, and single jobs)
+
+---
+
+<span id="execution-modes"></span>
+
+# Execution modes changes
+
+Workloads can be run as single jobs, sessions, or in a batch:
+
+- Use **sessions** mode for iterative workloads.
+- Use **batch** mode to submit multiple primitive jobs simultaneously.
+- Use **job** mode to submit a single primitive request without a context manager.
+
+
+## Best practices
+
+To ensure the most efficient use of the execution modes, the following practices are recommended:
+
+- Always close your session, either by using a context manager or by specifying `session.close()`.
+- There is a fixed overhead associated with running a job. In general, if your each of your job uses less than one minute of QPU time, consider combining several into one larger job (this applies to all execution modes). "QPU time" refers to time spent by the QPU complex to process your job.
+
+    A job's QPU time is listed in the **Usage** column on the IQP [Jobs](https://quantum.ibm.com/jobs) page, or you can  query it by using  `qiskit-ibm-runtime` with this command `job.metrics()["usage"]["quantum_seconds"]`
+
+- If each of your jobs consume more than one minute of QPU time, or if combining jobs is not practical, you can still run multiple jobs in parallel. Every job goes through both classical and quantum processing. While a QPU can process only one job at a time, up to five classical jobs can be processed in parallel. You can take advantage of this by submitting multiple jobs in [batch](#divide) or [session](#two-vqe) execution mode.
+
+The above are general guidelines, and you should tune your workload to find the optimal ratio, especially when using sessions. For example, running smaller jobs in parallel might be more cost effective because of reduced wall clock time.
```

Comment:
Is this statement true? If running a session, the wall clock time will be reduced but the cost is likely to be higher because sessions are billed on both classical and quantum time.

Reply:
> Is this statement true? If running a session, the wall clock time will be reduced but the cost is likely to be higher because sessions are billed on both classical and quantum time.

I got this from @jyu00

Reply:
> Is this statement true? If running a session, the wall clock time will be reduced but the cost is likely to be higher because sessions are billed on both classical and quantum time.

Since sessions will be billed on wall clock time (after it's activated), then it should cost less if wall clock is reduced, right?

Reply:
That seems reasonable -- maybe my statement was from something old.